### PR TITLE
Add cheat sheet links in PyDPF-Core doc

### PR DESCRIPTION
**Pull request by PipKat**
This PR adds the standard "Doc and Issues" section in the README file and overall index.rst file for the doc. It also cleans up these two pages so that they are basically consistent with the corresponding files for PyDPF-Core.

### CODE REVIEW

**Comment by PipKat** [comment]
@PProfizi This has the failed Codacy Static Code Analysis check, but all other checks have passed. Would you please review, accept, and merge? Thanks.

**Comment by PipKat** [comment]
@ansys/pyansys-core Any chance someone could review and merge this PR? Paul P. said that a PR can be merged with this failure, but no one has reviewed and accepted it yet.

**Review by Revathyvenugopal162 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -4,44 +4,61 @@
...
-  - CFX (`.cad/dat.cff`, `.flprj`)
+- Mechanical APDL (`.rst`, `.mode`, `.rfrq`, `.rdsp`)
+- LS-DYNA (`.d3plot`, `.binout`)
+- Fluent (`.cas/dat.h5`, `.flprj`)
+- CFX (`.cad/dat.cff`, `.flprj`)
 
-See the `PyDPF-Core main page <https://dpf.docs.pyansys.com/version/stable/index.html>`_
-for more information on compatibility.
+For more information on file support, see the [main page](https://dpf.docs.pyansys.com/version/stable/index.html)
+in the PDF-Core documentation.
```
Comment (Revathyvenugopal162) [inline README.md:25]:
```suggestion
in the PyDPF-Core documentation.
```

File: README.md [hunk 2]
```diff
@@ -4,44 +4,61 @@
...
+development version or previously released versions.
+
+You can also [view](https://cheatsheets.docs.pyansys.com/pydpf-post_cheat_sheet.png) or
+[download](https://cheatsheets.docs.pyansys.com/pydpf-post_cheat_sheet.pdf) the
+PyDPF-Post cheat sheet. This one-page reference provides syntax rules and commands
+for using PyDPF-Post.
+
+On the [PyDPF-Post Issues](https://github.com/ansys/pydpf-post/issues) page,
+you can create issues to report bugs and request new features. On the
+[PyDPF-Core Discussions](https://github.com/ansys/pydpf-post/discussions) page or the [Discussions](https://discuss.ansys.com/)
```
Comment (Revathyvenugopal162) [inline README.md:48]:
```suggestion
[PyDPF-Post Discussions](https://github.com/ansys/pydpf-post/discussions) page or the [Discussions](https://discuss.ansys.com/)
```

**Review by MaxJPRey — APPROVED** [review]
LGTM.

**Comment by MaxJPRey** [comment]
> @ansys/pyansys-core Any chance someone could review and merge this PR? Paul P. said that a PR can be merged with this failure, but no one has reviewed and accepted it yet.

All good to merge @PipKat even with this failure due to indentation.
Thanks!

**Pull request merged**